lib/components/dropdown.vue

```diff
@@ -28,7 +28,7 @@
             <span class="sr-only">{{toggleText}}</span>
         </b-button>
 
-        <div :class="['dropdown-menu',{'dropdown-menu-right': right}]"
+        <div :class="['dropdown-menu',{'dropdown-menu-right': right, show: visible}]"
```

Comment:
Not sure.  Will have to play with V4.beta in dev console to see if it i really needed